bindings/rust/s2n-tls/benches/handshake.rs

```diff
@@ -0,0 +1,22 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use s2n_tls::testing::s2n_tls::tests::{build_config, s2n_tls_pair};
+use criterion::{criterion_group, criterion_main, Criterion};
+
+pub fn handshake(c: &mut Criterion) {
+    let mut group = c.benchmark_group("s2n-tls");
+
+    group.bench_function("handshake_20200207", |b| {
```

Comment:
if testing all of the policies are going to look the exact same it might be worth putting it in a `for` loop over a list and creating the `bench_function` programmatically.